projects/angular-draggable-droppable/src/lib/droppable.directive.ts

```diff
@@ -121,10 +122,10 @@ export class DroppableDirective implements OnInit, OnDestroy {
           }
         );
 
-        let currentDragDropData: any;
+        let currentDragDropData: DragEvent;
```

Comment:
nit: maybe we could rename this from `currentDragDropData` to `currentDragEvent`?